.github/workflows/ci.yml

```diff
@@ -27,51 +27,41 @@ jobs:
           command: fmt
           args: --all -- --check
 
-  # Cross compilation is only needed for the main client and not the entire
-  # workspace.
-  cross-compile:
-    name: Cross Compile
+  docs:
+    name: Check Docs
     runs-on: ubuntu-latest
-    env:
-      CARGO_TARGET_ARMV7_UNKNOWN_LINUX_GNUEABIHF_LINKER: arm-linux-gnueabihf-gcc
-      CARGO_TARGET_ARM_UNKNOWN_LINUX_GNUEABIHF_LINKER: arm-linux-gnueabihf-gcc
-      CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER: aarch64-linux-gnu-gcc
-    strategy:
-      matrix:
-        target:
-          - aarch64-unknown-linux-gnu
-          - arm-unknown-linux-gnueabihf
-          - armv7-unknown-linux-gnueabihf
-        client:
-          - rspotify/cli,rspotify/env-file,rspotify/client-ureq,rspotify/ureq-rustls-tls,rspotify-http/client-ureq,rspotify-http/ureq-rustls-tls
-          - rspotify/cli,rspotify/env-file,rspotify/client-reqwest,rspotify/reqwest-rustls-tls,rspotify-http/client-reqwest,rspotify-http/reqwest-rustls-tls
     steps:
       - name: Checkout sources
         uses: actions/checkout@v2
 
-      - name: Install gcc for armhf
-        run: sudo apt-get update && sudo apt-get install gcc-arm-linux-gnueabihf gcc-aarch64-linux-gnu
-
       - name: Install stable toolchain
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: stable
           override: true
-          target: ${{ matrix.target }}
 
-      - name: Run cargo build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: -p rspotify -p rspotify-http -p rspotify-model -p rspotify-macros --no-default-features --features=${{ matrix.client }} --target ${{ matrix.target }} 
+      - name: Install cargo-deadlinks
+        run: cargo install cargo-deadlinks
+
+      - name: Build and Check
+        run: RUSTDOCFLAGS='-D warnings' cargo doc --features=cli,env-file
```

Comment:
Do we just check dead links for `cli` and `env-file` features, with ignoring other features?

Reply:
The rest of the features don't add anything new to the docs.